### PR TITLE
Add workspaces, admin

### PR DESCRIPTION
**Pull request by sboh1214**
image

### CODE REVIEW

**Review by buttercrab — CHANGES_REQUESTED** [review]
please run `npm i` before commit

**Review by buttercrab — DISMISSED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,41 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```
Comment (utilForever) [inline .gitignore]: I think this is unnecessary comment.

File: auth/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cubby_server_auth"
+version = "0.1.0"
+authors = ["Chris Ohk <utilforever@gmail.com>", "Seungoh Han <so.h4ns@gmail.com>", "Jaeyong Sung <jaeyong0201@gmail.com>", "Seungbin Oh <sboh1214@gmail.com>"]
```
Comment (utilForever) [inline auth/Cargo.toml]: We should add new members next time. 📝

File: game/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cubby_server_game"
+version = "0.1.0"
+authors = ["Chris Ohk <utilforever@gmail.com>", "Seungoh Han <so.h4ns@gmail.com>", "Jaeyong Sung <jaeyong0201@gmail.com>", "Seungbin Oh <sboh1214@gmail.com>"]
```
Comment (utilForever) [inline game/Cargo.toml]: We should add new members next time. 📝

File: middleware/Cargo.toml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cubby_server_middleware"
+version = "0.1.0"
+authors = ["Chris Ohk <utilforever@gmail.com>", "Seungoh Han <so.h4ns@gmail.com>", "Jaeyong Sung <jaeyong0201@gmail.com>", "Seungbin Oh <sboh1214@gmail.com>"]
```
Comment (utilForever) [inline middleware/Cargo.toml]: We should add new members next time. 📝

**Review by utilForever — DISMISSED** [review]
LGTM. Note that we should add new members next time. 🕊️

File: .gitignore (hunk 1 above)
Comment (sboh1214) [reply .gitignore]: Changed to what ignores for, who added them.

File: auth/Cargo.toml (hunk 2 above)
Comment (sboh1214) [reply auth/Cargo.toml]: Added Minchurl Shin <minchurl001@gmail.com>

File: game/Cargo.toml (hunk 3 above)
Comment (sboh1214) [reply game/Cargo.toml]: Added Minchurl Shin <minchurl001@gmail.com>

File: middleware/Cargo.toml (hunk 4 above)
Comment (sboh1214) [reply middleware/Cargo.toml]: Added Minchurl Shin <minchurl001@gmail.com>

**Review by utilForever — APPROVED** [review]
LGTM now. 🐦

**Review by buttercrab — APPROVED** [review]

**Pull request merged**